src/status_im2/contexts/wallet/common/account_switcher/view.cljs

```diff
@@ -17,8 +17,8 @@
     :or   {icon-name           :i/close
            accessibility-label :top-bar
            switcher-type       :account-options}}]
-  (let [{:keys [color emoji]} (rf/sub [:wallet/current-viewing-account])
-        networks              (rf/sub [:wallet/network-details])]
+  (let [{:keys [color emoji watch-only?]} (rf/sub [:wallet/current-viewing-account])
+        networks                          (rf/sub [:wallet/network-details])]
```

Comment:
any chance you could add the one liner for testnet in the :wallet/network-details sub here?

i.e 
_src/status_im2/subs/wallet/networks.cljs_
```
(re-frame/reg-sub
 :wallet/filtered-networks-by-mode
 :<- [:wallet/networks]
 :<- [:profile/test-networks-enabled?]
 (fn [networks test-networks-enabled?]
   (get networks (if test-networks-enabled? :test :prod))))
```
or should we do that in a separate pr?

Reply:
I would advise creating a separate PR as this sub is not used in any screens yet and can be quickly merged.

Reply:
Sounds good! 👌